### PR TITLE
[fix](ms_bvars) fix the units of bvar in meta service

### PR DESCRIPTION
**Pull request by spaces-X**
## Proposed changes

fix the value unit of g_bvar_ms_##func_name from us to ms

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by spaces-X** [comment]
run buildall

File: cloud/src/meta-service/meta_service_helper.h [hunk 1]
```diff
@@ -155,7 +155,7 @@ inline MetaServiceCode cast_as(TxnErrorCode code) {
         finish_rpc(#func_name, ctrl, response);                                          \
         closure_guard.reset(nullptr);                                                    \
         if (config::use_detailed_metrics && !instance_id.empty() && !drop_request) {     \
-            g_bvar_ms_##func_name.put(instance_id, sw.elapsed_us());                     \
+            g_bvar_ms_##func_name.put(instance_id, sw.elapsed_us() / 1000);              \
```
Comment (gavinchou) [inline cloud/src/meta-service/meta_service_helper.h:158]: the word "ms" of "g_bvar_ms" is not the time unit, it is short for meta-service.
we intend to use the time unit microsecond(us).

**Review by gavinchou — COMMENTED** [review]
No need to change, thank you.

File: cloud/src/meta-service/meta_service_helper.h (hunk 1 above)
Comment (spaces-X) [reply cloud/src/meta-service/meta_service_helper.h:158]: ok get it

**Pull request closed without merging**